lib/ramble/docs/application_dev_guide.rst

```diff
@@ -0,0 +1,16 @@
+.. Copyright 2022-2023 Google LLC
+
+   Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+   https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+   <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+   option. This file may not be copied, modified, or distributed
+   except according to those terms.
+
+.. _application-dev-guide:
+
+============================
+Application Developers Guide
+============================
+
+To be written....
```

Comment:
Shall we mark this with an easily greppable TODO or something if we're going to leave it unpopulated?

Reply:
Added!